src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -265,9 +270,13 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
             </p>;
         } else if (this.state.joinRule === JoinRule.Invite) {
             publicPrivateLabel = <p>
-                { _t(
-                    "Only people invited will be able to find and join this room.",
-                ) }
+                { _t("Only people invited will be able to find and join this room.") }
+                &nbsp;
+                { _t("You can change this at any time from room settings.") }
+            </p>;
+        } else if (this.state.joinRule === JoinRule.Knock && this.knockingEnabled) {
+            publicPrivateLabel = <p>
+                { _t("Anyone can knock on this room to join.") }
```

Comment:
Designs seems to suggest a different string - see https://www.figma.com/file/7TqjqdMBaPpm3IavKsMYu6/%5BBWI%5D-Knocking?node-id=367%3A57810